src/pages/patientView/patientHeader/GenomeDrivenDiagnosis.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from "react";
+import DefaultTooltip from "shared/components/defaultTooltip/DefaultTooltip";
+import {placeArrowBottomLeft} from "shared/components/defaultTooltip/DefaultTooltip";
+import SimpleTable from "shared/components/simpleTable/SimpleTable";
+import FontAwesome from "react-fontawesome";
+
+export type GDDOutput = {
+    Pred1: string;
```

Comment:
Why not `pred1` (all lowercase variable name)? Would it be better two have 2 arrays here instead of 6 variables, something like:

```
export type GDDOutput = {
    preds: string[];
    confs: string[]
}
```

Also, maybe use the full names `prediction` and `confidence` instead of truncated versions `pred` and `conf` for clarity?

Reply:
@onursumer this is the data format as returned from GDD.  We could transform it for esthetic purposes, though not sure if that adds confusion